### DISCUSSION TITLE
Can't install dusk, dompdf and phpspreadsheet  on laravel 5.5

### DISCUSSION THREAD

**Discussion by kalieg** [laravel/framework#36696] [Q&A]
Hi,
I am new to the Laravel frame work and I am trying to install packages using the composer update command in Laravel 5.5.
But at the end I got the following error messages:

- Downloading laravel/dusk (v2.0.14)
Failed to download laravel/dusk from dist: curl error 16 while downloading https://api.github.com/repos/laravel/dusk/zipball/a7aedd50dd5b4965ee589087ba575cb03981fe80: Error in the HTTP2 framing layer
Now trying to download from source

- Downloading dompdf/dompdf (v0.8.6)
Failed to download dompdf/dompdf from dist: curl error 28 while downloading https://codeload.github.com/dompdf/dompdf/legacy.zip/db91d81866c69a42dad1d2926f61515a1e3f42c5: Unexpected EOF
Now trying to download from source

 - Downloading phpoffice/phpspreadsheet (1.12.0)
 -   Failed to download phpoffice/phpspreadsheet from dist: curl error 28 while downloading https://codeload.github.com/PHPOffice/PhpSpreadsheet/legacy.zip/f79611d6dc1f6b7e8e30b738fc371b392001dbfd: OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0
    Now trying to download from source

Any help is highly appreciated!!!

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.
